### PR TITLE
GET request for Get Teams (User Groups) endpoint

### PR DESCRIPTION
**Pull request by shogath**
Hey!
I implemented a GET request for [Get Teams (User Groups)](https://clickup.com/api/clickupreference/operation/GetTeams1/) endpoint.

Here is the result of "example/user-groups/main.go"
clickup-user-groups

### CODE REVIEW

**Comment by w-haibara** [comment]
Note: Get Teams (`/group`)
> This endpoint is used to view [Teams: user groups](https://docs.clickup.com/en/articles/4010016-teams-how-to-create-user-groups) in your Workspace.
>
> In our API documentation, team_id refers to the id of a Workspace, and group_id refers to the id of a user group.

https://clickup.com/api/clickupreference/operation/GetTeams1/

File: clickup/user_groups.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	Email          string `json:"email"`
+	Color          string `json:"color"`
+	Initials       string `json:"initials"`
+	ProfilePicture string `json:"profilePicture"`
+}
+
+type GetUserGroupsOptions struct {
+	TeamID  string `url:"team_id,omitempty"`
+	GroupID string `url:"group_id,omitempty"`
+}
```
Comment (w-haibara) [inline clickup/user_groups.go:37]: Is this parameter (`group_id`) correct? According to the documentation, `group_ids` may be correct.

> Enter one or more Team ids (user groups) to retrieve information about specific Teams.
https://clickup.com/api/clickupreference/operation/GetTeams1/

```suggestion
type GetUserGroupsOptions struct {
	TeamID  string `url:"team_id,omitempty"`
	GroupIDs string `url:"group_ids,omitempty"`
}
```

File: clickup/user_groups.go [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+func (s *UserGroupsService) GetUserGroups(ctx context.Context, opts *GetUserGroupsOptions) ([]UserGroup, *Response, error) {
+	u := "group"
+	u, err := addOptions(u, opts)
```
Comment (w-haibara) [inline clickup/user_groups.go:41]:
```suggestion
	u, err := addOptions("group", opts)
```

**Review by w-haibara — COMMENTED** [review]
Hi, @shogath ! Thanks for the PR!
I'm glad you added the test and the exempt code.
I made a few comments. Please check them out.

File: clickup/user_groups.go (hunk 1 above)
Comment (shogath) [reply clickup/user_groups.go]: Hey! Yes, you are correct, I missed it. Fixed it.

**Comment by w-haibara** [comment]
LGTM!!
Thank you 🎉

**Review by w-haibara — APPROVED** [review]
LGTM!!
Thank you 🎉

**Pull request merged**